lib/spack/llnl/util/tty/__init__.py

```diff
@@ -7,10 +7,12 @@
 import fcntl
 import os
 import struct
+import subprocess
```

Comment:
I tried using `spack.util.executable` instead, but that file imports `llnl.util.tty`, creating a circular import dependency.

Reply:
This is good -- stuff in `llnl` shouldn't have dependencies on anything in `spack`